modules/nw-bgp-examples.adoc

```diff
@@ -0,0 +1,267 @@
+// Module included in the following assemblies:
+//
+// * networking/bgp-routing/about-bgp-routing.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-bgp-examples_{context}"]
+= Examples advertising pod IP addresses with BGP
+
+The following examples describe several different configurations for advertising pod IP addresses with BGP. The external network border router has the `172.18.0.5` IP address.
+
+Each example relies upon the following `FRRConfiguration` object, which OVN-Kubernetes generated from a `RouteAdvertisements` CR:
+
+.`FRRConfiguration` CR generated by OVN-Kubernetes
+[source,yaml]
+----
+apiVersion: frrk8s.metallb.io/v1beta1
+kind: FRRConfiguration
+metadata:
+  creationTimestamp: "2024-06-11T14:22:37Z"
+  generation: 1
+  name: metallb-ovn-worker
+  namespace: metallb-system
```

Comment:
I believe the namespace where `FRRConfiguration` CRs now have to be created under is `openshift-network-operator`. @fedepaol or @jcaamano should be able to confirm.